src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+}
+
+.stars__star--first {
+  margin-left: 0;
+}
+
+.stars__star--active {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
```suggestion
```
redundant. already settled

Reply:
If I delete these lines, the stars are repeated, and it looks wrong (in a container there are one and a half stars)